src/main/java/dev/djoon/racingcar/ui/ResultView.java

```diff
@@ -0,0 +1,19 @@
+package dev.djoon.racingcar.ui;
+
+public class ResultView {
+  public static void printNewGame() {
+    System.out.println("실행 결과");
+  }
+
+  public static void printXPos(int xPos) {
+    System.out.print("-"); /** 기본 출발선(=0) **/
+    for (int i=0; i<xPos; i++) {
```

Comment:
연산자 앞뒤로 한 칸씩 공백을 만들어주세요.
```suggestion
    for (int i = 0; i < xPos; i++) {